js/ui/templates/top-blocked-truncated.es6.js

```diff
@@ -1,5 +1,5 @@
 const bel = require('bel')
-const listItems = require('./shared/top-blocked-list-items.es6.js')
```

Comment:
Since we're using a different ux for the main popup card, I thought I'd take it out in another template.
Both top tracker list item templates are now out of the shared/ dir